scripts/create_general_thread_for_old_comments.rb

```diff
@@ -0,0 +1,16 @@
+Post.unscoped.each do |post|
```

Comment:
**suggestion** (performance): filter out the posts we don't need to check in SQL

```ruby
Post.unscoped.where(id: Comment.unscoped.where(comment_thread: nil)
                               .select(Arel.sql('distinct post_id'))).each do |post|
```